cmd/migrate_storage.go

```diff
@@ -25,13 +27,13 @@ import (
 var CmdMigrateStorage = cli.Command{
 	Name:        "migrate-storage",
 	Usage:       "Migrate the storage",
-	Description: "This is a command for migrating storage.",
+	Description: "This is a command for migrating current storage to external storage.",
```

Comment:
```suggestion
	Description: "This is a command to migrate between different storage types.",
```

Reply:
I mean this command will migrate some type of Gitea storage out of Gitea. It's not migrating from external storage.

Reply:
```go
		cli.StringFlag{
			Name:  "storage, s",
			Value: "",
			Usage: "New storage type: local (default) or minio",
		},
		cli.StringFlag{
			Name:  "path, p",
			Value: "",
			Usage: "New storage placement if store is local (leave blank for default)",
		},
```
This tells something different.

Reply:
Yeah, I mean `New storage` is the external storage, it's not the same as the storage configured in app.ini. The storage configured in app.ini is the source storage, `New storage` or external storage is the destination storage.

Reply:
But if the current storage is extern (min.io or something) you can migrate to a local storage?

Reply:
I think yes. Previously users are using minio, but now they want to convert back to local storage. Then we could support that via this command.

Reply:
So that's why I think the current description `This is a command for migrating current storage to external storage.` is wrong.

Reply:
How about `This is a command for migrating Gitea configured storage to a parameter indicated storage.`

Reply:
Changed in 404a32beb86bc6cf2845025be32b5a434bab216b

Reply:
I think that's a very complicated description but that should someone decide who is a native english speaker.

Reply:
> I think that's a very complicated description but that should someone decide who is a native english speaker.

@zeripath Help needed.